lib/Steps/OpenSentry.ts

```diff
@@ -29,7 +30,16 @@ export class OpenSentry extends BaseStep {
 
       BottomBar.hide();
 
-      const urlToOpen = `${baseUrl}account/settings/wizard/${data.hash}/`;
+      let urlToOpen = `${baseUrl}account/settings/wizard/${data.hash}/`;
+      if (this._argv.signup) {
+        urlToOpen += '?signup=1';
+        // integration maps to platform in the wizard
+        if (this._argv.integration) {
+          urlToOpen += `&project_platform=${mapIntegrationToPlatform(
+            this._argv.integration,
+          )}`;
+        }
+      }
```

Comment:
This is fine for now I think but should this grow in complexity let's use [`new URL` + `URLSearchParams`](https://nodejs.org/api/url.html#class-urlsearchparams)

Reply:
@lforst is that even available with node 16?